src/main/java/com/google/devtools/build/lib/skyframe/PrerequisiteProducer.java

```diff
@@ -124,13 +126,16 @@ static class State implements SkyKeyComputeState, IncompatibleTargetProducer.Res
      * <p>Non-null only while the computation is in-flight.
      */
     @Nullable private Driver incompatibleTargetProducer;
+
     /**
      * If a value is present, it means the target was directly incompatible.
      *
      * <p>Non-null after the {@link #incompatibleTargetProducer} completes.
```

Comment:
(If you update things, this comment might need to be updated).

Reply:
Done